### PR TITLE
Refactor: Prefix CSS Classes

### PR DESCRIPTION
**Pull request by varun13169**
Refactored the following component.

1) Badge Component
2) Button Component
3) Card Component
4) Alert Component
5) Avatar Component

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]

**Comment by bhushannaikaroskar** [comment]
All good varun. Seems ok to me

**Review by bhushannaikaroskar — APPROVED** [review]

File: components/button/button.html [hunk 1]
```diff
@@ -32,14 +32,14 @@ <h1 class="comp-lib-h1">Button Component</h1>
     <h2 class="comp-lib-h2">Primary Button</h2>
     <div class="component_demo_container">
         <!-- Button Component Starts -- Primary -->
-        <button class="btn btn--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs reset-button-inherit-parent">
+        <button class="dui-btn dui-btn--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs reset-button-inherit-parent">
```
Comment (manojsarna) [inline components/button/button.html:35]: please look into this , so many classes for a single button 🤯

File: components/button/button.html [hunk 2]
```diff
@@ -51,14 +51,14 @@ <h2 class="comp-lib-h2">Primary Button</h2>
     <h2 class="comp-lib-h2">Link Button</h2>
     <div class="component_demo_container">
         <!-- Link Button Component Starts -- Link Primary -->
-        <a class="link link--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs" href="#">
+        <a class="dui-link dui-link--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs" href="#">
```
Comment (manojsarna) [inline components/button/button.html:54]: if you can please refractor the number of classes

File: library/card/card.css [hunk 3]
```diff
@@ -88,70 +88,70 @@
...
 
-.card__img--basic-with-dismiss-button {
+.dui-card__img--basic-with-dismiss-button {
     object-fit: cover;
     width: 100%;
     height: 100%;
 }
 
-.card__actions--basic-with-dismiss-button  {
+.dui-card__actions--basic-with-dismiss-button  {
```
Comment (manojsarna) [inline library/card/card.css:97]: class names are too long

**Review by manojsarna — COMMENTED** [review]
please look into the reviews

File: components/button/button.html (hunk 1 above)
Comment (varun13169) [reply components/button/button.html:35]: Multiple classes are for standardizing Typography, Boarder-Radius, and Spacing as per Figma screen.

File: components/button/button.html (hunk 2 above)
Comment (varun13169) [reply components/button/button.html:54]: Yup will be consolidating in further iterations.

File: library/card/card.css (hunk 3 above)
Comment (varun13169) [reply library/card/card.css:97]: Yup will be consolidating in further iterations.

**Review by iRohitGaur — COMMENTED** [review]
good to merge 👍🏻

**Pull request merged**